src/vs/workbench/parts/terminal/electron-browser/terminalConfigHelper.ts

```diff
@@ -96,7 +96,7 @@ export class TerminalConfigHelper implements ITerminalConfigHelper {
 		return DEFAULT_ANSI_COLORS[baseThemeId];
 	}
 
-	private _measureFont(fontFamily: string, fontSize: number, lineHeight: number): ITerminalFont {
+	private _measureFont(fontFamily: string, fontSize: number, lineHeight: number, fontWeight: string): ITerminalFont {
```

Comment:
It's fine to leave out font weight from font measurement as the terminal assumes we're working with a monospace font where bold size is the same as non-bold size.